cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -86,11 +74,26 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate rules
     ).convert { it.expandTilde() }
         .file(mustExist = false, canBeFile = false, canBeDir = true, mustBeWritable = false, mustBeReadable = false)
         .convert { it.absoluteFile.normalize() }
+        .outputGroup()
 
     private val outputFormats by option(
         "--output-formats", "-f",
         help = "The list of output formats to be used for the ORT result file(s)."
-    ).enum<FileFormat>().split(",").default(listOf(FileFormat.YAML))
+    ).enum<FileFormat>().split(",").default(listOf(FileFormat.YAML)).outputGroup()
+
+    private val rules by mutuallyExclusiveOptions(
```

Comment:
Shouldn't this code block move actually have happened in the previous commit?

Reply:
Initially I saw that as an input option, only while grouping I realized it would make sense to have a separate group for this option. I could move that to the previous commit, but I don't see much value in it.

Reply:
I agree, and was just curious.